win-agent/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM microsoft/nanoserver:10.0.14393.1593
+ENV RANCHER_AGENT_WINDOWS_IMAGE rancher/win-agent:v0.0.1
+RUN mkdir "c:/program files/rancher"
+VOLUME [ "c:/program files/rancher" ]
+ADD [ "http://dev-tool.oss-cn-shenzhen.aliyuncs.com/windows/devcon.exe","c:/program files/" ]
```

Comment:
It seems bad to be downloading third party binaries. What does this binary do? Could we at least do an md5sum check on the binary? If we do that, you might have to move downloading the binary and md5suming it to the build-image script.

Reply:
That is a tool for handling device on windows. There is no way I can download from Microsoft alone. So I put it into cloud storage.